judo-runtime-core-dao-rdbms/src/main/java/hu/blackbelt/judo/runtime/core/dao/rdbms/AbstractRdbmsDAO.java

```diff
@@ -226,6 +248,16 @@ public void delete(EClass eClass, ID id) {
         }
     }
 
+    @Override
+    @SneakyThrows(SQLException.class)
+    public void deleteAll(EClass eClass, Iterable<ID> ids) {
+        try (MetricsCancelToken ct = getMetricsCollector().start(METRICS_DAO_QUERY)) {
+            List<ID> idList = new ArrayList<>();
+            ids.forEach(idList::add);
+            deletePayload(eClass, idList);
+        }
+    }
```

Comment:
Why does `deleteAll` need metrics collector if `createAll` and `updateAll` do not?

Reply:
Because createAll and updateAll use the create and update functions in which MetricsCollector is instantiated, but deleteAll calls the deletePayload function in which MetricsCollector is not instantiated.